ecommerce/auth/json_token.py

```diff
@@ -18,6 +19,6 @@ def create_access_token(subject: str):
 
 def create_refresh_token(subject: str):
     expire_time = datetime.utcnow() + timedelta(minutes=REFRESH_TOKEN_EXPIRE_MINUTES)
-    to_encode = {"sub": subject, "exp": expire_time, "type": "access_token"}
+    to_encode = {"sub": subject, "exp": expire_time, "type": "refresh_token"}

```

Comment:
```suggestion
    to_encode = {"jti": uuid4(), "sub": subject, "exp": expire_time, "type": "refresh_token"}
```
Add unique token identifier?